cmd/controller-manager/app/options/options.go

```diff
@@ -37,6 +37,7 @@ const (
 	defaultMaxRequeueNum       = 15
 	defaultSchedulerName       = "volcano"
 	defaultHealthzAddress      = ":11251"
+	defaultListenAddress       = ":8080"
```

Comment:
Would a different port be better?
1. Make a distinction with the scheduler port.
2. Port 8080 is too common and prone to conflicts.